marimo/_plugins/ui/_impl/dataframes/transforms/apply.py

```diff
@@ -75,6 +76,12 @@ def get_handler_for_dataframe(
         if isinstance(df, pl.DataFrame):
             return PolarsTransformHandler()
 
+    if DependencyManager.ibis.has():
+        import ibis
+
+        if isinstance(df, ibis.expr.types.Table):
```

Comment:
`ibis.Table` will be more robust to internal refactors, since it's publicly exposed in our `api` module.